Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4144,21 +4234,29 @@ bool IWindowExplorer.AddSwitchWindowAction(string Title)
 
         void IWindowExplorer.HighLightElement(ElementInfo ElementInfo)
         {
-            unhighlightLast();
+            UnhighlightLast();
 
-            if (string.IsNullOrEmpty(ElementInfo.XPath))
-            {
-                Driver.SwitchTo().DefaultContent();
-                SwitchFrame(ElementInfo.Path, ElementInfo.XPath, true);
-                ElementInfo.XPath = GenerateXpathForIWebElement((IWebElement)ElementInfo.ElementObject, "");
-            }
-            else
+            Driver.SwitchTo().DefaultContent();
+            SwitchFrame(ElementInfo.Path, ElementInfo.XPath, true);
+
```

Comment:
here we using the ElementInfo.XPath value but only in later code we checks if this need to be calculated, something wrong here